.github/workflows/issue-pr-triage.yml

```diff
@@ -0,0 +1,34 @@
+name: Auto-triage to project boards
+
+on:
+  issues:
+    types: [opened, labeled]
+  pull_request:
+    types: [opened, labeled]
+
+jobs:
+  content_project:
+    runs-on: ubuntu-latest
+    name: Triage to Content Project
+    steps:
+    - name: Triages NEW issues and NEW pull requests to the Content Project
+      uses: srggrs/assign-one-project-github-action@1.2.0
+      if: |
+        contains(github.event.issue.labels.*.name, 'content') ||
+        contains(github.event.pull_request.labels.*.name, 'content')
+      with:
+        project: 'https://github.com/newrelic/docs-website/projects/3'
+        column_name: 'Needs Triage'
+
+  localization_project:
+    runs-on: ubuntu-latest
+    name: Triages to Localization Project
+    steps:
+    - name: Triages NEW issues and NEW pull requests to the Localization Project
+      uses: srggrs/assign-one-project-github-action@1.2.0
+      if: |
+        contains(github.event.issue.labels.*.name, 'localization') ||
+        contains(github.event.pull_request.labels.*.name, 'localization')
+      with:
+        project: 'https://github.com/orgs/newrelic/projects/47/'
```

Comment:
Does the trailing slash matter. Now that we've been bit with trailing slash, I notice this everywhere 😄

Reply:
Github URLS work with both a slash and no slash, is there a preference?

Reply:
Nope! Just wanted to make sure this would still work :)